pravega/schema-registry
# Issue 38: Creating helm charts for schema registry deployment

**Change log description**  
Provides helm charts for deployment of schema registry on a Kubernetes cluster.

**Purpose of the change**  
Fixes #38 

**What the code does**  
Simplifies deployment of schema-registry for end users through the use of helm.

**How to verify it**  
The following command should successfully deploy schema-registry on your cluster
```
helm install <release-name> charts/schema-registry --set controllerUri='tcp://<controller-ip>:9090'
```
